text/0000-machine-readable-tests-output.md

```diff
@@ -0,0 +1,146 @@
+- Feature Name: machine\_readable\_tests\_output
+  - Start Date: 2015-09-17
+  - RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Replace current test output with machine-readable one and add thin compatibility
+layer on top of that.
```

Comment:
I mean display. I don't know how tooling should be done. There should be external program that will parse output and display it in the same way it is. Maybe it should be included into `rustc` but maybe into `cargo`. I have no idea and this need to be resolved.